axonserver/src/main/java/io/axoniq/axonserver/localstorage/EventStorageEngine.java

```diff
@@ -182,8 +184,9 @@ void processEventsPerAggregateHighestFirst(String aggregateId, long minSequenceN
     /**
      * Adds information to the actuator health endpoint for this event store.
      * @param builder actuator health builder
+     * @param diskSpaceHealthIndicatorProperties properties or health indicators
      */
-    default void health(Health.Builder builder) {
+    default void health(Health.Builder builder, DiskSpaceHealthIndicatorProperties diskSpaceHealthIndicatorProperties) {
```

Comment:
These properties should not be part of the health operation, but set on the EventStorageImplementation level.